src/actions/ens/getEnsText.ts

```diff
@@ -62,16 +65,19 @@ export type GetEnsTextReturnType = string | null
  * })
  * // 'wagmi_sh'
  */
-export async function getEnsText<TChain extends Chain | undefined>(
+export async function getEnsText<
+  TChain extends Chain | undefined,
+  TKeys extends string | string[],
```

Comment:
Usually we try to use `readonly` arrays if we can.

```suggestion
  TKeys extends string | readonly string[],
```